schema/rdl_schema_0.1.json

```
@@ -48,20 +50,24 @@
 			"format": "date"
 		},
 		"version": {
+			"title": "Dataset version",
 			"type": "integer",
-			"description": "Version of the dataset"
+			"description": "The version of the dataset."
```

Suggest aligning with DCAT's [`version`](https://www.w3.org/TR/vocab-dcat-3/#Property:resource_version):
```suggestion
			"description": "The version indicator (name or identifier) of the dataset."
```